scripts/inline_comments_data_collection.py

```diff
@@ -184,11 +188,24 @@ def process_comments(patch_threshold, diff_length_threshold):
 def main():
     os.makedirs("patches", exist_ok=True)
     os.makedirs("dataset", exist_ok=True)
-    dataset_file_path = "dataset/inline_comment_dataset2.json"
+
+    dataset_file_path = "dataset/inline_comment_dataset.json"
+    dataset_url = "https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.data_comments.latest/artifacts/public/inline_comment_dataset.json.zst"
+    dataset_version = 1
+
     with open(dataset_file_path, "a") as dataset_file_handle:
         for data in process_comments(patch_threshold=1000, diff_length_threshold=5000):
             dataset_file_handle.write(json.dumps(data) + "\n")
 
+    register(
+        path=dataset_file_path,
+        url=dataset_url,
+        version=dataset_version,
+        support_files=[],
+    )
+
+    upload(path=dataset_file_path)
```

Comment:
I do not think that you need to use `upload()`. TC will upload the files that you specify in `data-pipeline.yml`.

Reply:
Removed here: 2e0d6e0dff28ab74e41f49708f2a756168630b37